README.md

```diff
@@ -98,24 +96,20 @@ Save them to your `.bash_profile` or equivalent.
   * Any errors that pop up are probably from missing modules or missing environmental variables.
     Read the errors!
 
-### Just check to see some very simple checking if a point falls within a shape in the DB
-1.  Visit http://server.ip/zonecheck
-
 ### Adding new data
 
-*Probably need to put this stuff in its own document eventually, but for now it can live here.  Much of this should also be automatable; whatever isn't needs to be explained more clearly.  For now it's notes-to-self.*
+*Hooray!  Much of the process is now automated.  Still TODO: make this documentation clearer and add suggestions for less standardised content.*
 
-0. Export a shapefile with an attribute that can be used to look up appropriate snugget text and intensity values for each shape. It can be in any SRS, and it doesn't matter if there are additional attributes, but be aware that those will be lost in the import.
+0. Export a shapefile with an attribute that can be used to look up appropriate snugget text and intensity values for each shape. It can be in any SRS, and it doesn't matter if there are additional attributes, but be aware that those will be lost in the import. *Note that if you use multiple shapefiles that cover different areas (e.g. some cover a county and some cover a whole state), users will see partial results without a warning that there's missing data. For clearest results, crop all your shapefiles to the same area.*
```

Comment:
Definitely.  I think at some point a broader rewrite of these instructions will make sense.  In my mind, it's 3 sections:
* Automated process for simple, regular data (in which we explain what snuggets are and how to format that CSV file)
* How to work around this with the shapefiles if you have more complex data and some GIS skills
* How to manually build on the existing Django code structure if you'd rather add more complex data that way